example/example.cpp

```diff
@@ -2,12 +2,17 @@
 // Distributed under the MIT software license, see the accompanying
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
+#include <init.capnp.h>
+#include <init.capnp.proxy.h>
+
+#include <cstring>
```

Comment:
IWYU's suggestion here appears to be incorrect.

Reply:
re: https://github.com/bitcoin-core/libmultiprocess/pull/184#discussion_r2166969762

Added keep pragma here too